tools/mqppep/mqppep_anova.xml

```diff
@@ -0,0 +1,227 @@
+<tool
+  id="mqppep_anova"
+  name="MaxQuant Phosphopeptide ANOVA"
+  version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@"
+  python_template_version="3.5"
+  profile="21.05"
+  >
+    <description>Perform ANOVA on merged and filtered data from phospho-peptide enrichment/MaxQuant pipeline</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <!--
+      The weird invocation used here is because knitr and install_tinytex
+      both need access to a writeable directory, but most directories in a
+      biocontainer are read-only, so this builds a pseudo-home under /tmp
+    -->
+    <command detect_errors="exit_code"><![CDATA[
+      cp '$__tool_directory__/mqppep_anova_script.Rmd' . || exit 0;
+      cp '$__tool_directory__/mqppep_anova.R'          . || exit 0;
+      Rscript mqppep_anova.R
+        --inputFile '$input_file'
+        --alphaFile '$alpha_file'
+        --firstDataColumn $intensity_column_regex_f
+        --imputationMethod $imputation.imputation_method
+        #if $imputation.imputation_method == "random"
+          --meanPercentile '$imputation.meanPercentile'
+          --sdPercentile   '$imputation.sdPercentile'
+        #end if
+        --regexSampleNames $sample_names_regex_f
+        --regexSampleGrouping $sample_grouping_regex_f
+        --imputedDataFile $imputed_data_file
+        --imputedQNLTDataFile '$imp_qn_lt_file'
+        --reportFile '$report_file';
+      export RESULT=\$?;
+      exit \${RESULT}
+    ]]></command>
+    <configfiles>
+      <configfile name="sample_names_regex_f">
+        $sample_names_regex
+      </configfile>
+      <configfile name="sample_grouping_regex_f">
+        $sample_grouping_regex
+      </configfile>
+      <configfile name="intensity_column_regex_f">
+        $intensity_column_regex
+      </configfile>
+    </configfiles>
+    <inputs>
+        <param name="input_file" type="data" format="tabular" label="Filtered Phosphopeptide Intensities"
+               help="[input_file] Phosphopeptide intensities filtered for minimal quality.  First column label 'Phosphopeptide'; sample-intensities must begin in column 10 and must have column labels to match argument [sample_names_regex]"
```

Comment:
what are those [] in the help? The Parameter is called `--inputFile` so you can use argument="--inputFile" remove the name= and remove the []

Reply:
okay, I will adjust the parameter names and eliminate the cues to the users (which I have been doing in imitation of the W4M practices).

Reply:
Where is this documented? I guess we need to change it.

Reply:
I guess it only really helps and makes the annotation of parameters better if we use argument="" with the real underlying argument, isn't it?